gloria_config.yml

```diff
@@ -5,21 +5,21 @@
 # indicated. All inputs are case-sensitive.
 
 local_settings:
-- data_dir: "" # point to the *full* directory path where your location information is stored - this path must end with a '/'
-- location_file: "" # name of the *.csv* file that contains the location information
-- unique_id: "" # this is the column that stores the unique identifier for each site, should not contain any special characters
-- latitude: "" # this is the column that stores the latitude of the site, must be in decimal degrees
-- longitude: "" # this is the column that stores the longitude of the site, must be in decimal degrees
-- location_crs: "" # this is the coordinate reference system of the location data, must be in EPSG format (e.g. EPSG:4326)
-- polygon: "" # optional. True or False - if True, you have lake polygon shapefiles for each site.
+- data_dir: "/Users/steeleb/Documents/GitHub/Landsat_C2_SR_GLORIA/data/" # point to the *full* directory path where your location information is stored - this path must end with a '/'
+- location_file: "collated_locations_for_pull.csv" # name of the *.csv* file that contains the location information
+- unique_id: "rowid" # this is the column that stores the unique identifier for each site, should not contain any special characters
+- latitude: "Latitude" # this is the column that stores the latitude of the site, must be in decimal degrees
+- longitude: "Longitude" # this is the column that stores the longitude of the site, must be in decimal degrees
+- location_crs: "EPSG:4326" # this is the coordinate reference system of the location data, must be in EPSG format (e.g. EPSG:4326)
+- polygon: "False" # optional. True or False - if True, you have lake polygon shapefiles for each site.
```

Comment:
Does this need to exist anymore, if the targets for pulling from polygons has been removed?

Reply:
good find! Nope, definitely does not need to exist anymore! I'll edit now.